configs/dynamic_rcnn/dynamic_rcnn_r50_fpn_1x.py

```diff
@@ -0,0 +1,112 @@
+_base_ = '../faster_rcnn/faster_rcnn_r50_fpn_1x_coco.py'
```

Comment:
If the parameter is the same as the one in `_base_`, we could omit it so that the config will look clean. 
Duplicate components like `neck`, `RPNHead` could be omitted. 
You may take `configs/double_heads/dh_faster_rcnn_r50_fpn_1x_coco.py` as an example.

Reply:
Fixed.